packages/utilities/src/utilities/StickyHeader/StickyHeader.js

```diff
@@ -13,6 +13,9 @@ import settings from 'carbon-components/es/globals/js/settings';
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+const c4dPrefix = 'c4d';
+const prefixV2 = 'cds';
```

Comment:
The name `prefixV2` reads as ambiguous when it's all by itself below. Should it just be `cdsPrefix`?